Makefile

```diff
@@ -838,6 +838,7 @@ LIB_OBJS += base85.o
 LIB_OBJS += bisect.o
 LIB_OBJS += blame.o
 LIB_OBJS += blob.o
+LIB_OBJS += bloom.o
```

Comment:
Commit message: you mention the `--bloom` option. Replace with `--changed-paths`.